public/Lab-Files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title> Blog </title>
+</head>
+<body>
+    <h1>Yo-ho yo-ho a blogger's life for me</h1>
+    <p>zyxwvutsrqponmlkjihgfedcba. Zyxwvutsrqponmlkjihgfedcba. <br> 
+    zyxwvutsrqponmlkjihgfedccba</p>
+</body>
+</html>
```

Comment:
Great job on this challenge. 
Your indentation is awesome too. The code is easy to read.